### DISCUSSION TITLE
mini.starter only suggest recents in $PWD

### DISCUSSION THREAD

**Discussion by beaumccartney** [nvim-mini/mini.nvim#751] [Q&A] [answered]
by default, it seems that mini.starter just suggests the most recent files I visited (or something like that). could I filter the suggestions such that only files in whatever directory i'm in (or children of that dir)

**Comment by echasnovski** [comment, marked as answer]
Yes. You can set [`current_dir` argument to `true` in `sections.recent.files()`](https://github.com/echasnovski/mini.nvim/blob/9968f6e221ae7bdac57a910c5bf2026186aa023c/doc/mini-starter.txt#L367-L368). This requires building your own items. [Here](https://github.com/echasnovski/mini.nvim/blob/9968f6e221ae7bdac57a910c5bf2026186aa023c/doc/mini-starter.txt#L88-L103) is an example.
